### PR TITLE
feat(css): Add data for `stop-color` & `stop-opacity`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Additional details

https://svgwg.org/svg2-draft/pservers.html#StopColorProperty
https://svgwg.org/svg2-draft/pservers.html#StopOpacityProperty

### Related issues and pull requests

/cc @lahmatiy 

Relates to https://github.com/mdn/data/issues/750
Relates to https://github.com/csstree/csstree/issues/50

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -9461,6 +9461,38 @@
...
+    ],
+    "initial": "black",
+    "appliesto": "stopElements",
+    "computed": "asSpecified",
+    "order": "perGrammar",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/stop-color"
+  },
+  "stop-opacity": {
+    "syntax": "<opacity-value>",
```
Comment (skyclouds2001) [inline css/properties.json:9481]:
```suggestion
    "syntax": "<'opacity'>",
```

File: css/syntaxes.json [hunk 2]
```diff
@@ -566,6 +566,9 @@
   "opacity()": {
     "syntax": "opacity( [ <number-percentage> ] )"
   },
+  "opacity-value": {
+    "syntax": "<number> | <percentage>"
+  },
```
Comment (skyclouds2001) [inline css/syntaxes.json:571]:
```suggestion
```

File: css/properties.json [hunk 3]
```diff
@@ -9461,6 +9461,38 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"
   },
+  "stop-color": {
+    "syntax": "<color>",
```
Comment (skyclouds2001) [inline css/properties.json:9465]:
```suggestion
    "syntax": "<'color'>",
```

**Review by pepelsbey — APPROVED** [review]
Looks good! Thank you :)

**Pull request merged**